### DISCUSSION TITLE
Return number of clicks remaining in API response?

### DISCUSSION THREAD

**Discussion by shrimpchip28** [YOURLS/YOURLS#3105] [General]
Is there a way to return the remaining clicks left in the api response? the number of clicks is returned but I don't believe the remaining number is returned. 

*** UPDATE. Disregard. My bad. Answered my own question. I completely overlooked the 'countdown' key returned.

**Comment by dgw** [comment]
Note: This feature comes from [a plugin](https://github.com/joshp23/YOURLS-Expiry), not YOURLS core.

**Reply by shrimpchip28** [reply]
Thanks. It's been a while since I looked at my implementation and jumped the gun on this post.  Thanks for the quick response.
